packages/accordion/src/plugin/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { YooptaPlugin } from '@yoopta/editor';
+import { AccordionElementKeys, AccordionListItemProps } from '../types';
+import { AccordionList } from '../renders/AccordionList';
+import { AccordionItem } from '../renders/AccordionItem';
+import { AccordionItemHeading } from '../renders/AccordionItemHeading';
+import { AccordionItemContent } from '../renders/AccordionItemContent';
+
+const Accordion = new YooptaPlugin<AccordionElementKeys, AccordionListItemProps>({
+  type: 'Accordion',
+  elements: {
+    'accordion-list': {
+      asRoot: true,
+      render: AccordionList,
+      children: ['accordion-list-item'],
+    },
+    'accordion-list-item': {
+      render: AccordionItem,
+      children: ['accordion-list-item-heading', 'accordion-list-item-content'],
```

Comment:
wondering an issue if you mistype the 'key', or if they will be the same within one plugin for multiple items